### PR TITLE
add more documentation, add make command for running devstack server

### PR DESCRIPTION
**Pull request by adampalay**
@peter-fogg and @clintonb ,

I added some more documentation for getting started with using ecommerce on devstack. I also added a make command to run the ecommerce server on devstack, just to make it a little easier to do so.

### CODE REVIEW

**Comment by adampalay** [comment]
@awaisdar001 , @aamir-khan , FYIs

File: Makefile [hunk 1]
```diff
@@ -39,6 +40,9 @@ migrate:
 serve:
 	python manage.py runserver 0.0.0.0:8002
 
+devserve:
```
Comment (peter-fogg) [inline Makefile:43]: Nice, good idea.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -88,10 +88,12 @@ OIDC client via LMS admin.
...
     $ python manage.py runserver 8002
 
-If you're running on devstack, you'll need to pass the appropriate settings:
+If you're running on devstack, you'll need to pass the appropriate settings
+after switching to the ecommerce user:
 
 .. code-block:: bash
 
+    $ sudo su ecommerce
     $ python manage.py runserver 0.0.0.0:8002 --settings=ecommerce.settings.devstack
```
Comment (peter-fogg) [inline docs/getting_started.rst]: This would be a good place to mention `make devserve`.

Comment (adampalay) [inline docs/getting_started.rst]: ah, good call

**Comment by adampalay** [comment]
@peter-fogg , updated

**Comment by peter-fogg** [comment]
:shipit:

**Comment by adampalay** [comment]
@clintonb can you please take a look?

**Comment by clintonb** [comment]
:+1:

**Pull request merged**